thirdeye/thirdeye-frontend/tests/acceptance/self-serve-import-metric-test.js

```diff
@@ -0,0 +1,168 @@
+      'Navigation to import page succeeded');
+    assert.ok(
+      $(`${selfServeConst.SECONDARY_LINK}:contains(${returnLinkText})`).get(0),
+      'Return link appears.'
+    );
+    assert.ok(
+      $(`${selfServeConst.SECONDARY_LINK}:contains(${inGraphsLinkText})`).get(0),
+      'InGraph link appears.'
+    );
+  });
+
+  test(`visiting metric import page to import existing dashboard`, async (assert) => {
+    await visit(`/self-serve/import-metric`);
+
+    const $dashboardInput = $(selfServeConst.IMPORT_DASHBOARD_INPUT);
+    let $submitButton = $(selfServeConst.SUBMIT_BUTTON);
+
+    assert.ok(
+      $submitButton.get(0).disabled,
+      'Submit button is still disabled'
+    );
+    assert.ok(
+      $(selfServeConst.IMPORT_DASHBOARD_INPUT).length,
+      'Import dashboard field renders.'
+    );
+
+    await fillIn(selfServeConst.IMPORT_DASHBOARD_INPUT, 'test123X');
+
+    assert.notOk(
+      $submitButton.get(0).disabled,
+      'Submit button is enabled'
+    );
+
+    await click(selfServeConst.SUBMIT_BUTTON);
+
+    assert.equal(
+      $(selfServeConst.IMPORT_WARNING).get(0).innerText,
+      importWarning,
+      'Non-existent dashboard name results in warning'
+    );
+    assert.ok(
+      $submitButton.get(0).disabled,
+      'Submit button is now disabled.'
+    );
+
+    await fillIn(selfServeConst.IMPORT_DASHBOARD_INPUT, dashboardToImport);
+    await triggerKeyEvent(selfServeConst.IMPORT_DASHBOARD_INPUT, 'keyup', '8');
+
+    assert.notOk(
+      $submitButton.get(0).disabled,
+      'Submit button is enabled'
+    );
+
+    await click(selfServeConst.SUBMIT_BUTTON);
+
+    assert.ok(
+      $(selfServeConst.IMPORT_SUCCESS).get(0).innerText.includes(importSuccessMsgText1),
+      'Import dashboard successful - success text displays)'
+    );
+    assert.ok(
+      $(selfServeConst.IMPORT_RESULT_LIST).children().length === 5,
+      'At least one metric was imported'
+    );
+    assert.equal(
+      $(selfServeConst.SUBMIT_BUTTON).get(0).innerText,
+      'Onboard Another Dashboard',
+      'Submit button mode changed correctly'
+    );
+    assert.ok(
+      $dashboardInput.get(0).disabled,
+      'Custom fields are disabled'
+    );
+
+    await click(selfServeConst.SUBMIT_BUTTON);
+
+    assert.equal(
+      $(selfServeConst.SUBMIT_BUTTON).get(0).innerText,
+      'Import Metrics',
+      'Submit button mode changed correctly'
+    );
+    assert.notOk(
+      $dashboardInput.get(0).disabled,
+      'Custom fields are now enabled again.'
+    );
+  });
+
+  test(`visiting metric import page to import custom dashboard`, async (assert) => {
+    await visit(`/self-serve/import-metric`);
+
+    const rrdName = 'my-new-rrd';
+    const metricName = 'my-new-metric';
+    const datasetName = 'my-new-dataset';
+    const importSuccessMsgText2 = `You have successfully onboarded the ${datasetName} dataset , including the following metrics`;
+    let $submitButton = $(selfServeConst.SUBMIT_BUTTON);
```

Comment:
same, does this need to be a `let`?

Reply:
Same here. Need to re-cache the element to look at its new state.